client/src/components/SiteSearch/SiteSearchListing/SiteSearchListing.module.scss

```diff
@@ -36,9 +36,6 @@
   flex-direction: column;
   flex-grow: 1;
 }
-.container thead {
-  display: none;
-}
```

Comment:
__Suggestion__
Rather than add CSS (more code) to hide unnecessary content added by JavaScript (undesired code), try to not output the content at all. Maybe there is a way with ReactTable(?) to not render the headers (less code).

Reply:
I'm not sure if I understand, but it might help if I explain what I'm doing here. In [this mockup ](https://xd.adobe.com/view/db2660cc-1011-4f26-5d31-019ce87c1fe8-ad17/screen/448bf6f3-cb39-4d19-b3f2-0660b9af7dc0/) I saw that the header bar should be showing (without labels), so I removed this piece of CSS which was hiding it. This should display a horizontal bar above the listing without any labels.

Reply:
Great, thanks. Never mind my comment. I wasn't at my most lucid when writing this.